### PR TITLE
Bug 1802420, clarify whether PKI injection is only available to operators

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1802420

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@danehans Thanks again for your guidance! PTAL.

@xltian Who is the best contact to provide a QE review? Thanks!

**Comment by ahardin-rh** [comment]
QE approved in the BZ

File: modules/certificate-injection-using-operators.adoc [hunk 1]
```diff
@@ -25,3 +25,39 @@ The Operator mounts this ConfigMap into the container's local trust store.
 Adding a trusted CA certificate is only needed if the certificate is not
 included in the {op-system-first} trust bundle.
 ====
+
+Certificate injection is not limited to Operators. The Cluster Network Operator
+is responsible for fulfilling the certificate injection request and will do so
```
Comment (kalexand-rh) [inline modules/certificate-injection-using-operators.adoc]: Maybe s/is responsible for fulfilling the certificate injection request and will do so/injects certificates

**Review by kalexand-rh — COMMENTED** [review]
I might make one tweak, but this LGTM

**Pull request merged**